### PR TITLE
Add details card to reports sub tab.

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

**Comment by Ron-Lavi** [comment]
@amirfefer should we close it and someone else will take over?

File: webpack/src/Extends/Host/PuppetTab/SubTabs/Reports/index.js [hunk 1]
```diff
@@ -28,6 +40,13 @@ const Reports = ({ hostName }) => (
 
 Reports.propTypes = {
   hostName: PropTypes.string.isRequired,
+  hostInfo: PropTypes.object,
```
Comment (Ron-Lavi) [inline webpack/src/Extends/Host/PuppetTab/SubTabs/Reports/index.js:43]: Note to myself, reports tab has moved to core which mean we can import it directly and remove slot&fill in this case, cc @nadjaheitmann @amirfefer

**Comment by Ron-Lavi** [comment]
continue the work from this PR in https://github.com/theforeman/foreman_puppet/pull/274

**Comment by nadjaheitmann** [comment]
> continue the work from this PR in #274

Shall we close this PR, then?

**Comment by Ron-Lavi** [comment]
> > continue the work from this PR in #274
> 
> Shall we close this PR, then?

yep, let's close it :)

**Pull request closed without merging**